sunray/comm.cpp

```diff
@@ -1053,6 +1056,12 @@ void mqttCallback(char* topic, byte* payload, unsigned int length) {
   }
 }
 
+// define a macro so avoid repetitive code lines for sending single values via MQTT
+#define MQTT_SEND(VALUE, FORMAT, TOPIC) \
```

Comment:
Perhaps a little more meaningful in the mqtt context:

```suggestion
#define MQTT_PUBLISH(VALUE, FORMAT, TOPIC) \
```